packages/api/endpoints/collections.js

```diff
@@ -86,10 +87,10 @@ async function get(req, res) {
   const version = req.params.version;
 
   try {
-    const c = new models.Collection();
-    const result = await c.get({ name, version });
-    // const stats = await collection.getStats([res], [res.name]);
```

Comment:
Whoa - so we were requesting stats that were never returned?

Reply:
No - this was commented out.   

IIRC, originally I think the collections endpoint was using the models.stats to return stats with each collection call, but this was wisely refactored due to API latency to a separate stats.js endpoint and/or other alls.

Reply:
The rest of this change is just models.Collection.get -> collectiongPgModel.get.